SmartEVSE2.X/EVSE.c

```diff
@@ -2600,7 +2601,7 @@ void main(void) {
                             for (x = 0; x < 3; x++) {
                                 // Calculate difference of Mains and PV electric meter
                                 if (PVMeter) CM[x] = CM[x] - PV[x];             // CurrentMeter and PV resolution are 1mA
-                                Irms[x] = CM[x]/100;                            // reduce resolution of Irms to 100mA
+                                Irms[x] = (int)CM[x]/100;                       // reduce resolution of Irms to 100mA
```

Comment:
This will not work, as CM(x) is reduced to an int before the division.

Reply:
I can confirm it *does* work, and it fails to work without this cast on my SolarEdge CM. I will investigate further, and maybe spell out the casting more explicitly where the casting issue would otherwise occur.

Reply:
IMHO it should be (int)(CM[x]/100)